packages/web/components/ui/equation-node.tsx

```
@@ -51,25 +50,25 @@ export function EquationElement(props: PlateElementProps<TEquationElement>) {
             className={cn(
               'group flex cursor-pointer items-center justify-center rounded-sm select-none hover:bg-primary/10 data-[selected=true]:bg-primary/10',
               props.element.texExpression.length === 0
                 ? 'bg-muted p-3 pr-9'
-                : 'px-2 py-1',
+                : 'px-2 py-1'
             )}
             data-selected={selected}
             contentEditable={false}
-            role="button"
+            role='button'
           >
```

Biome (1.9.4)

[error] The HTML element with the interactive role 'button' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'button'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)